docs/DeviceCtl/Definitions.md

```diff
@@ -0,0 +1,10 @@
+# Definitions
+
+<strong>Device</strong> - A single entity composed of physical ports, such as a switch or switch stack, which can be polled in aggregate via a single interface.
```

Comment:
Use Markdown and nto HTML please, so `**` instead of `strong`

Reply:
:+1: done, also replaced strong in some older files.

Reply:
Perfect, thanks